qtop.py

```diff
@@ -314,9 +331,9 @@ def fill_cpucore_columns(state_np_corejob, cpu_core_dict, id_of_username, max_np
     np = state_np_corejob['np']
     corejobs = state_np_corejob.get('core_job_map', '')
 
-    if state == '?':
-        for cpu_line in cpu_core_dict:
-            cpu_core_dict[cpu_line] += '_'
+    if state == '?':  # for non-existent machines
+        for core_line in core_user_map:
+            core_user_map[core_line] += ['#']  # was '_'. @Fotis Maybe closer to reality? (No machine, no cores!)
```

Comment:
drop comment; can't recall the why of this; we'll be coming back to it, so make it a config tunable?